tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import os
+from collections import defaultdict
+from typing import Any, DefaultDict, Dict, List, Tuple, Union
+
+import numpy as np
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Catalog, Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import chunked
+
+DATASET_NAME = "VGGFace2"
+_SEGMENTS = {"train": True, "test": False}
```

Comment:
The values of `_SEGMENTS` seems useless, consider use `Tuple` instead?

Reply:
I use segments at first because it seems that only the experimental group have some attributes(used in box2d), but I don't confirm  it is true, so I ignore the condition whether it is the experimental group, and directly  the dictionary to judge whether it exists or not